dashboard/config/scripts/ECSPD3-Unit2.script

```diff
@@ -33,7 +33,7 @@ level 'ECSPD U2 D 4-6'
 level 'ECSPD U2 D 7-9 Intro'
 level 'ECSPD U2 D 10-12 Intro'
 level 'ECSPD Binary Odometer'
-level 'ECSPD Why Binary - The Internet Video'
+level 'U1L5 How the Internet Works - Video'
```

Comment:
Yep, I talked to Brook about this one.  The `ECSPD Why Binary - The Internet Video` level had the exact same video as `U1L5 How the Internet Works - Video`, but with extra text for the teacher.

I moved the text to `teacher_markdown` so it now shows up in a blue box below the video (for teachers).